### ISSUE TITLE
Is it even possible to build tensorflow on Windows? 

### ISSUE THREAD

**Issue by ammarsalman94** [tensorflow/tensorflow#52141]
**System information**
- Windows 10 Pro x64 - 19043.1237
- TensorFlow installed from (source or binary): source
- TensorFlow version: 2.4
- Python version: 3.8
- Installed using virtualenv? pip? conda?: no
- Bazel version (if compiling from source): 3.7.1
- GCC/Compiler version (if compiling from source): VC 14.29.30133
- CUDA/cuDNN version: 11.0/8.0
- GPU model and memory: Nvidia GTX 1080

I checked the the build from source page and followed the instructions. I have the repository cloned, I checked out 2.4 and I configured it with default settings except for 'cuda support' which I enabled. 

I go to bazel, try to build it, and all hell breaks loose: 

`bazel build -c opt --config=cuda --define=no_tensorflow_py_deps=true --copt=-nvcc_options=disable-warnings //tensorflow:tensorflow.dll --verbose_failures.` 

> ERROR: Config value 'cuda' is not defined in any .rc file

Even if I give up on cuda option, it still fails: 

`bazel build -c opt --define=no_tensorflow_py_deps=true --copt=-nvcc_options=disable-warnings //tensorflow:tensorflow.so --verbose_failures`

> ERROR: no such target '//tensorflow:tensorflow.dll': target 'tensorflow.dll' not declared in package 'tensorflow'

Then I tried a suggestion to use libtensorflow_cc.so: 

`bazel build -c opt --define=no_tensorflow_py_deps=true --copt=-nvcc_options=disable-warnings //tensorflow:libtensorflow_cc.so --verbose_failures`

> ERROR: no such target '//tensorflow:libtensorflow_cc.so': target 'libtensorflow_cc.so' not declared in package 'tensorflow'

In other words, nothing is even remotely working. 

I don't understand anything about anything at this point. Is it not supposed to work? Was support for Windows removed?

**Comment by sushreebarsa** [comment]
@ammarsalman94 Could you please try to Build from [Source ](https://www.tensorflow.org/install/source_windows#setup_for_windows) using TF latest stable version 2.6.0 and have a look at the similar issue [link](https://stackoverflow.com/questions/34785414/how-to-install-tensorflow-on-windows) , Please let us know if it helps? Thanks!

**Comment by ammarsalman94** [comment]
I tried with version 2.6, same issues. I looked at the SO answer provided, in summary it says Bazel doesn't work on Windows and I have to build for Linux. If I wanted to run Tensorflow on Linux I wouldn't be trying to build it for Windows. So that defeats the whole purpose. Is that the conclusion? It simply doesn't build on Windows and the whole instruction pages are pretty much a waste of time?

**Comment by Saduf2019** [comment]
@ammarsalman94 
As per [guide](https://www.tensorflow.org/install/source_windows) it is possible to have bazel on windows, please verify the system [requirements](https://www.tensorflow.org/install/pip#system-requirements) and hardware requirements. [refer to [link](https://github.com/tensorflow/tensorflow/issues/52143#issuecomment-927651685])]

**Comment by Saduf2019** [comment]
@ammarsalman94 
Thank you for your response, glad the issue is resolved.
